master/internal/webhooks/postgres_webhook.go

```diff
@@ -39,6 +40,13 @@ type WebhookManager struct {
 	expToWebhookConfig map[int]*expconf.WebhooksConfigV0
 }
 
+// CustomTriggerData is the data for custom trigger.
+type CustomTriggerData struct {
+	Title       string `json:"title"`
+	Description string `json:"description"`
+	Level       string `json:"level"`
```

Comment:
nit: Should we keep the `CustomWebhookEventData` fields in the same order as this `CustomTriggerData`?
Just so it's clear what the most important field(s) are (which is presumably the `Title` field)